libs/metrics/Cargo.toml

```diff
@@ -1,11 +1,12 @@
 [package]
 name = "metrics"
 version = "0.1.0"
-edition = "2021"
+edition.workspace = true
 license = "Apache-2.0"
```

Comment:
The same is possible for license. Will you update more fields?

Reply:
> The same is possible for licens

I don't see it possible for `license` field, only for `license-file`, that's different.

Reply:
https://github.com/EmbarkStudios/rust-gpu/blob/main/crates/rustc_codegen_spirv-types/Cargo.toml

You showed me this example

Reply:
Indeed that works, switched.